site/content/integrate/apps/quickstart/quick-start-serverless/_index.md

```diff
@@ -30,32 +30,15 @@ See complete source code of the example [here](https://github.com/mattermost/mat
 
 ## Prerequisites
 
-Before you can start with your app, you first need to set up a local developer environment following the [server]({{< ref "/contribute/more-info/server/developer-setup" >}}) and [webapp]({{< ref "/contribute/more-info/webapp/developer-setup" >}}) setup guides. You must enable the apps feature flag before starting the Mattermost server by setting the environment variable `MM_FEATUREFLAGS_AppsEnabled` to `true` by e.g. adding `export MM_FEATUREFLAGS_AppsEnabled=true` to your `.bashrc` or using `make run-server MM_FEATUREFLAGS_AppsEnabled=true`.
+Before you can start with your app, you first need to set up a local developer environment following the [server]({{< ref "/contribute/more-info/server/developer-setup" >}}) and [webapp]({{< ref "/contribute/more-info/webapp/developer-setup" >}}) setup guides.
 
-In the System Console, ensure that the following are set to **true**:
+Once set up, open the System Console and ensure that the following are set to **true**:
 
 - `Enable Bot Account Creation`
 - `Enable OAuth 2.0 Service Provider`
 
 You also need at least `go1.16` installed. Please follow the guide [here](https://golang.org/doc/install) to install the latest version.
```

Comment:
```suggestion
You also need at least `go1.18` installed. Please follow the guide [here](https://golang.org/doc/install) to install the latest version.
```

Reply:
If we make this version change here, we should make it in all quick start guides.

Reply:
I agree, @neflyte! @levb - If you agree as well, would you be open to updating the remaining Go references for consistency and clarity?

Reply:
Maybe I'm missing something. Why is go needed for the other quickstart guides?